velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -268,27 +270,165 @@ TpchPlan TpchQueryBuilder::getQ6Plan() const {
   return context;
 }
 
-const std::vector<std::string> TpchQueryBuilder::kTableNames_ = {"lineitem"};
+TpchPlan TpchQueryBuilder::getQ18Plan() const {
+  static const std::vector<std::string> kTableNames = {
+      "lineitem", "orders", "customer"};
+  std::vector<std::string> lineitemSelectedColumns = {
```

Comment:
nit: consider shortening variable names for readability: lineitemColumns, ordersColumns, customerColumns.

Reply:
I think the values clearly show that this is a subset of all columns. Overall, shorter test would allow one to read it better and capture more of these details.